lib/def/def.ml

```diff
@@ -22,26 +24,14 @@ type ('a, 'b) choice =
 (** Alias to [Adef.cdate] *)
 type cdate = Adef.cdate
 
-(** Alias to [Adef.date] *)
```

Comment:
I think I liked the mutually recursive definition better. Even if recursivity is not needed.